external/model-preparation-algorithm/configs/instance-segmentation/efficientnetb2b_maskrcnn/template.yaml

```diff
@@ -40,8 +40,9 @@ hyper_parameters:
       stat_requests_number:
         default_value: 2
     nncf_optimization:
+      visible_in_ui: false
       enable_quantization:
-        default_value: true
+        default_value: false
```

Comment:
I'm not sure if default_value: false will disable NNCF task. Please, check it on the Geti side. As far as I remember, we removed nncf entrypoint: `mpa_tasks.apis.detection.DetectionNNCFTask` to disable the nncf task.

Reply:
Done, I'll check results on a SC instance